spec/features/saml/saml_logout_spec.rb

```diff
@@ -53,8 +52,7 @@
 
         # contains all redirect_uris in content security policy
         expect(page.response_headers['Content-Security-Policy']).to include(
-          'form-action \'self\' example.com example.com/ example.com/auth/result '\
-          'example.com/logout',
+          'form-action \'self\' http://example.com',
```

Comment:
what if we changed the quotes style on this?
```suggestion
          "form-action 'self' http://example.com",
```